Eastern-Research-Group/mywaterway
# Enhancement/729 move configs to s3

## Related Issues:
* [HMW-729](https://jira.epa.gov/browse/HMW-729)

## Main Changes:
* Moved most of the remaining config items to S3.

## Steps To Test:
* Community Page
  * Focus on waterbodies (overview and identified issues tab), monitoring locations (overview and monitoring tabs) and cyan (overview and monitoring tabs.
  * Test surrounding layers as well.
    * Test popups for surrounding USGS Streamgages. We will need to test the listviews once the issues with that service are resolved.
  * For monitoring locations test filtering.
*  Waterbody Report and Plan Summary pages
  * Test popups for waterbodies and surrounding monitoring locations.
* Monitoring Report
  * Test popups for monitoring locations and surrounding monitoring locations.
* State/Tribal Pages
  * Test popups for waterbodies, monitoring locations, and surrounding layers.
  * Navigate to http://localhost:3000/state/AL/water-quality-overview
  * Select "Lakes and Reservoirs" for the "Water Type"
  * Expand "Top Reaons for Impairment..."
  * Verify all rows are linked to the glossary (particularly "Acidity")

## TODO:
- [x] Will need to test USGS Stream Gauges, once that issue is resolved.
